internal/rollup/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "node_modules_none",
+)
+
+nodejs_binary(
+    name = "worker",
+    data = [
+        "worker.js",
+        "@npm//@bazel/worker",
```

Comment:
Do we need to make the `@npm` part configurable? I think previously the `rollup_bin` rule attribute allowed that?